pycontw/pycon.tw
# Re-organize the main menu and remove links from index

## Types of changes
Please put an `x` in the box that applies


## Description
**Describe what the change is**
Reorganize the main menu structure:
Events/Keynotes --> Conference/Keynotes
Events/Talks --> Conference/Talks
Events/Tutorials --> Conference/Tutorials
(Hidden) --> Conference/Community Track
Conference/Overview --> Events/Overview
Hide Sponsor

And also hide the huge button links from index (Talks / Become a volunteer / Sponsor)

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to index (localhost:8000)
2. Check that the menu is the same as what I stated in the description
3. Click on the links and see if menu is highlighted correctly

## More Information
**Screenshots**
![圖片](https://user-images.githubusercontent.com/484883/91330889-9ffaa580-e7fc-11ea-890b-627f1f06068a.png)
![圖片](https://user-images.githubusercontent.com/484883/91330943-b56fcf80-e7fc-11ea-9017-f59ac4299af3.png)
![圖片](https://user-images.githubusercontent.com/484883/91331006-cc162680-e7fc-11ea-8194-956c0f70f5b6.png)
![圖片](https://user-images.githubusercontent.com/484883/91331130-f2d45d00-e7fc-11ea-9057-4a2f5af727ad.png)
